web/cobrands/northamptonshire/base.scss

```diff
@@ -15,13 +15,27 @@
     width: 190px;
     height: 41px;
     padding: 11px 0;
-    background: url(/cobrands/northamptonshire/img/new-ncc-logo.png) 0 50% no-repeat;
+    background: url(/cobrands/northamptonshire/img/NorthAndWestLogo.png) 0 50% no-repeat;
+    background-size: contain;
 }
 
+#nav-link {
```

Comment:
I think you can replace all of this custom `#nav-link` styling with just `$menu-image: 'menu-black';` in `web/cobrands/northamptonshire/_colours.scss` (replacing the existing `$menu-image: 'menu-white';`).